src/Symfony/Bridge/Twig/Resources/views/Form/form_table_layout.html.twig

```diff
@@ -42,3 +42,14 @@
         {{- form_rest(form) -}}
     </table>
 {%- endblock form_widget_compound -%}
+
+{%- block empty_row -%}
+    {%- set translation_domain = translation_domain ?: 'sf_form' -%}
```

Comment:
It was a suggestion that we might provide default translations for "No choices..", "No Fields" and even the actual "None" as default placeholder like it's done for validation message, there is already the dependency.

I can remove it of courses :)